theano/sandbox/rng_mrg.py

```diff
@@ -425,19 +493,28 @@ def c_code(self, node, name, inp, out, sub):
         const npy_int32 MASK2 = 65535;      //2^16 - 1
         const npy_int32 MULT2 = 21069;
 
+        // The +1 is to avoid odims[0] which fails on windows
+        // We have to read size[i] as an int64, but odims has to be intp*
+        // for NumPy on 32-bit platforms.
```

Comment:
As you now do a malloc, you can remove the +1 and the first line of this comment.